### PR TITLE
draft a new blog: ORAS 0.14 and Future: Empower Container Secure Supply Chain

### PR DESCRIPTION
**Pull request by FeynmanZhou**
This PR is still in WIP. Please do not review or merge.

### CODE REVIEW

File: docs/blog/oras-0.14-and-future.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+Prior to ORAS CLI v0.14 release, the ORAS Go library, also released v2.0.0-rc.2 to support [artifacts-spec v1.0.0-rc.2](https://github.com/oras-project/artifacts-spec/releases/tag/v1.0.0-rc.2) and provide new functions to enable developers to build your own OCI client tool.
+
+As cloud native development continues to grow, we have seen increased community interest in evolving registries to natively store, pull, copy, and discover a graph of supply chain artifacts. Artifact references are important for many use cases such as adding Software Bill of Materials (SBoM), security scan results, and container image signing. 
+
+This blog will demonstrate how to use ORAS v0.14 to copy an image from a validated artifact registry to personal conintainer registry, then attach a SBOM to it and discover the graph.
+
+
+## Install ORAS 0.14
+
+Install the latest release of ORAS on a Linux machine:
```
Comment (SteveLasker) [inline docs/blog/oras-0.14-and-future.md:16]: This solves linux, should we point to a common install oras page in oras.land?
And, when, oh when can we apt-get/win-get install :)

File: docs/blog/oras-0.14-and-future.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+mv oras-install/oras /usr/local/bin/
+rm -rf oras_0.14.1_*.tar.gz oras-install/
+```
+
+## Copy an image from Registry A to Registry B
+
+In this demo, assume all images in are validated in MAR, so I will use ORAS to copy the container image from Microsoft Artifact Registry (MAR) to my personal repository of Docker Hub. You can use your prefered container registry with ORAS.
+
+```
+./oras copy mcr.microsoft.com/mmlspark/spark2.4:1.0.0 registry-1.docker.io/pengfeizhou/spark2.4:1.0.0
```
Comment (SteveLasker) [inline docs/blog/oras-0.14-and-future.md:31]: using `./oras` assumes it's in the same directory.
Can we change this to `oras copy` assuming the developer centrally installed it?

Comment (SteveLasker) [reply docs/blog/oras-0.14-and-future.md:31]: You shouldn't need registy-1 in the docker.io reference.
This should work
`./oras copy mcr.microsoft.com/mmlspark/spark2.4:1.0.0 docker.io/pengfeizhou/spark2.4:1.0.0`

File: docs/blog/oras-0.14-and-future.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+```
+curl -Lo sbom-tool https://github.com/microsoft/sbom-tool/releases/latest/download/sbom-tool-linux-x64
+chmod +x sbom-tool
+```
+
+Generate a SBOM for the Spark image stored in Docker Hub:
+
+```
+./sbom-tool generate -di registry-1.docker.io/pengfeizhou/spark2.4:1.0.0 -b ./foo -pn bar -pv 0.1 -bc ./foo -ps MyCompany -nsb http://mycompany.com
```
Comment (SteveLasker) [inline docs/blog/oras-0.14-and-future.md:48]: remove ./sbom-tool as it should install centrally
remove registry-1 from docker.io reference

File: docs/blog/oras-0.14-and-future.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+```
+
+## Attach the SBOM to this image
+
+Attach the generate SBOM to this Spark image stored in Docker Hub:
+
+```
+oras attach registry-1.docker.io/pengfeizhou/spark2.4:1.0.0 foo/_manifest/spdx_2.2/manifest.spdx.json --artifact-type example/sbom
```
Comment (SteveLasker) [inline docs/blog/oras-0.14-and-future.md:56]: love the `attach` command
Same comment about removing registry-1 from the docker.io reference.

**Review by SteveLasker — COMMENTED** [review]
Thanks @FeynmanZhou, excellent to see docs and blog posts, in addition to the awesome enhancements and release. 

A few nits before LGTM

File: docs/blog/oras-0.14-and-future.md (hunk 3 above)
Comment (FeynmanZhou) [reply docs/blog/oras-0.14-and-future.md:48]: There is a bug related to this: https://github.com/oras-project/oras/issues/542
I will update it to `docker.io` after the 0.15 release.

Comment (SteveLasker) [reply docs/blog/oras-0.14-and-future.md:48]: Oh, that's unfortunate. 
On the first reference to registry-1, can you add: See https://github.com/oras-project/oras/issues/542 for removing registry-1?

File: docs/blog/oras-0.14-and-future.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
+# ORAS 0.14 and Future: Empower Container Secure Supply Chain
+
+_Feynman Zhou, CNCF Ambassador, Product Manager_
+
+The [OCI Registry As Storage (ORAS)](https://oras.land/) project maintainers announced v0.14 release for the CLI tool on Aug 22. ORAS v0.14 provides four new top-level commands and new options to manage supply chain artifacts across different container registries and multi-cloud environments.
```
Comment (lachie83) [inline docs/blog/oras-0.14-and-future.md:5]:
```suggestion
The [OCI Registry As Storage (ORAS)](https://oras.land/) project maintainers recently announced v0.14 release for the CLI tool. ORAS v0.14 introduces four new top-level commands and new options to manage supply chain artifacts across different container registries and multi-cloud environments. 
```

File: docs/blog/oras-0.14-and-future.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
+# ORAS 0.14 and Future: Empower Container Secure Supply Chain
+
+_Feynman Zhou, CNCF Ambassador, Product Manager_
+
+The [OCI Registry As Storage (ORAS)](https://oras.land/) project maintainers announced v0.14 release for the CLI tool on Aug 22. ORAS v0.14 provides four new top-level commands and new options to manage supply chain artifacts across different container registries and multi-cloud environments. 
+
+Prior to ORAS CLI v0.14 release, the ORAS Go library, also released v2.0.0-rc.2 to support [artifacts-spec v1.0.0-rc.2](https://github.com/oras-project/artifacts-spec/releases/tag/v1.0.0-rc.2) and provide new functions to enable developers to build your own OCI client tool.
```
Comment (lachie83) [inline docs/blog/oras-0.14-and-future.md:7]:
```suggestion
Prior to ORAS CLI v0.14 release, the ORAS Go library, also released v2.0.0-rc.2 to support [artifacts-spec v1.0.0-rc.2](https://github.com/oras-project/artifacts-spec/releases/tag/v1.0.0-rc.2) and provides new functions to enable developers to build your own OCI client tool.
```

File: docs/blog/oras-0.14-and-future.md (hunk 2 above)
Comment (lachie83) [reply docs/blog/oras-0.14-and-future.md:31]: I would redact the destination registry name in this example

File: docs/blog/oras-0.14-and-future.md (hunk 5 above)
Comment (lachie83) [reply docs/blog/oras-0.14-and-future.md:5]: No need to commit my suggestions. You can add them directly to the source branch as I don't want to break DCO

**Pull request closed without merging**